torch/_dynamo/variables/builder.py

```diff
@@ -384,7 +384,7 @@ def index_source(key):
                     value=value,
                     guards=make_guards(GuardBuilder.CONSTANT_MATCH),
                 )
-        elif isinstance(value, frozenset) and (
+        elif istype(value, frozenset) and (
```

Comment:
This is a behavior change though as some people may rare subclass frozenset, right?